BenjixD/ProjectMooks
# Basic balancing and bug fixing

Project Mooks playtest 1&2:
- Fixed game crashing bugs
- Added !move functionality
- Change state algorithm to be based on League of Legends damange algorithm
- Enable timer for mooks (Currently only 15 seconds)
- Twitch Mic check
